src/ontology/mondo-edit.obo

```diff
@@ -556470,23 +556448,24 @@ is_a: MONDO:0005044 ! hypertensive disorder
 
 [Term]
 id: MONDO:0100079
-name: developmental and epileptic encephalopathy, 6
+name: developmental and epileptic encephalopathy, 6A
 def: "Any early infantile epileptic encephalopathy in which the cause of the disease is a mutation in the SCN1A gene." [OMIM:607208]
-comment: EEEI6 is the OMIM placeholder for SCN1A-related severe epilepsies. The vast majority of SCN1A-related severe epilepsies, including Dravet syndrome, are not early infantile by definition, thereby creating a oxymoronic conflict within the ontology. We recommend describing the phenotype, ie Dravet syndrome, or the genetic eitology, ie SCN1A to represent the concept in question. This is distinct from MONDO:0011794 Dravet syndrome. See https://github.com/monarch-initiative/mondo/issues/745
+comment: In Mondo, DEE6A is treated as a distinct class from Dravet syndrome (contrary to OMIM), as not every case of Dravet syndrome is caused by a variation in SCN1A. We recommend describing the phenotype, ie Dravet syndrome, or the genetic eitology, ie SCN1A to represent the concept in question. This is distinct from MONDO:0100135 Dravet syndrome. See https://github.com/monarch-initiative/mondo/issues/745
 subset: gard_rare {source="MONDO:GARD"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: rare
-synonym: "DEE6" EXACT ABBREVIATION []
-synonym: "developmental and epileptic encephalopathy, 6" EXACT CLINGEN_LABEL []
-synonym: "Dravet syndrome" RELATED []
+synonym: "DEE6A" EXACT ABBREVIATION []
+synonym: "developmental and epileptic encephalopathy, 6A" EXACT CLINGEN_LABEL []
+synonym: "Dravet syndrome" RELATED [OMIM:607208]
 synonym: "EIEE6" EXACT ABBREVIATION []
 synonym: "epileptic encephalopathy, early infantile, 6" EXACT [OMIM:607208]
 xref: OMIM:607208 {source="MONDO:equivalentTo"}
```

Comment:
According to the comment, OMIM should therefore not be an equivalentTo

Reply:
We discussed this yesterday - OMIM conflates Dravet with DEE6 but the epilepsy experts wanted Dravet to be a separate class b/c it is not always caused by this gene. So I guess this is a case where we are splitting a term and OMIM is lumping it.

If we make the OMIM MONDO:relatedTo, we won't have an equiv term to that OMIM ID, is that okay?